docs/website/docs/general-usage/schema.md

```diff
@@ -127,7 +127,46 @@ Postgres ignore it when creating tables.
 ### Variant columns
 
 Variant columns are generated by a normalizer when it encounters data item with type that cannot be
-coerced in existing column.
+coerced in existing column. Please see our [`coerce_row`](https://github.com/dlt-hub/dlt/blob/7d9baf1b8fdf2813bcf7f1afe5bb3558993305ca/dlt/common/schema/schema.py#L205) if you are interested to see how internally it works.
+
+Let's consider our [getting started](../getting-started#quick-start) example with slightly different approach,
+where `id` is an integer type at the beginning
+
+```py
+data = [
+  {"id": 1, "name": "Alice"}
+]
+```
+
+once pipeline runs we will have the following schema
+
+| name          | data_type     | nullable |
```

Comment:
Let's make "name" here more explicit so it doesn't clash with "name" the field below.